src/main/java/com/sample/QuestionWindow.java

```diff
@@ -13,6 +13,10 @@ public static String updateFrame(String questionValue, List<String> options) {
                 "Wybierz odpowiedz",
                 JOptionPane.DEFAULT_OPTION, JOptionPane.INFORMATION_MESSAGE, null, options.toArray(new String[options.size()]), options.get(0));
 		
-		return options.get(0);
+		return options.get(index);
+	}
+	
+	public static void showResults(String info) {
```

Comment:
Nie możemy wyświetlać zwykłym pytaniem?

Reply:
Możemy, ale nie mam pojęcia jak przekazać listę w locie. Zrobione na szybko, jeżeli masz lepsze rozwiązanie to się nie krępuj ;)